specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/examples/capacityReservationExamples/CapacityReservationGroup_CreateOrUpdate.json

```diff
@@ -23,7 +23,8 @@
               "id": "/subscriptions/{subscription-id2}"
             }
           ]
-        }
+        },
+        "reservationType": "Block"
```

Comment:
Pls. add new example files referencing block reservation type in CapacityReservationGroup and also for CapacityReservation created under such a group.